spec/models/user_spec.rb

```diff
@@ -25,4 +25,3 @@
     end
```

Comment:
- [ ] As requested of you by the previous reviewer, when I run ```rspec``` to check the unit tests in your spec folder, they are all failing. Kindly look into why this is the case.

![image](https://user-images.githubusercontent.com/86318284/208608837-b930d6ed-42ec-42b6-8d0a-769bb7fa117b.png)